tests/src/image-smoke.spec.ts

```diff
@@ -65,3 +68,20 @@ describe('Image pull verification', async () => {
     await playExpect(imageDetailPage.inspectTab).toBeVisible();
   });
 });
+
+describe('Image delete verification', async () => {
```

Comment:
We could probably change the first describe to a 'Image workflow verification', and then remove this, then unite two test methods into one?